# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: js/src/jit/LIROps.yaml
# Commit: 25adf505426d
# Full Hash: 25adf505426d21f67e15d146f879b0ea53d343a7
# Author: Ryan Hunt <rhunt@eqrion.net>
# Date: 2022-03-23 04:09:13
# Regressor Bug: 1642412
# File Overlap Count: 19
# Description:
#   Bug 1642412 - wasm: Rollup of inline table.get/set for anyref. r=lth
#   
#   Rollup of the previous inline table.get/set patches.
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D141024
# ==============================================================================

diff -r 43f7556db2e1 -r 25adf505426d js/src/jit/LIROps.yaml
--- a/js/src/jit/LIROps.yaml	Tue Mar 22 19:44:13 2022 +0000
+++ b/js/src/jit/LIROps.yaml	Tue Mar 22 19:59:05 2022 +0000
@@ -2723,9 +2723,18 @@
   arguments:
     offset: size_t
 
+- name: WasmLoadTableElement
+  result_type: WordSized
+  operands:
+    elements: WordSized
+    index: WordSized
+
 - name: WasmDerivedPointer
   gen_boilerplate: false
 
+- name: WasmDerivedIndexPointer
+  gen_boilerplate: false
+
 - name: WasmStoreRef
   operands:
     tls: WordSized